src/StatusQ/Components/StatusMemberListItem.qml

```diff
@@ -0,0 +1,64 @@
+import QtQuick 2.0
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core 0.1
+
+
+StatusListItem {
+    id: root
+
+    property string nickName: ""
+    property string userName: ""
+    property string keyChat: ""
+    property bool isMutualContact: false
+    property var trustIndicator: StatusMemberListItem.TrustedType.None
+    property string imageSource: ""
+    property bool isOnline: false
+
+    enum TrustedType {
```

Comment:
I think in StatusMessage we use Enum ContactType? Can we use similar enum so at to keep consistency when using this? You can also change it in StatusMessage, whatever you prefer.

Reply:
I wanted to open another issue for that as I was wondering that maybe we could share this enum and icons / colors definition however, I've realized that in `StatusMessage` it is not used the same implementation as here and I believe that it is a misunderstood from one or the other side because I understand it should be the same (or it is my interpretation of the implementation in `StatusMessage`, please correct me if I'm wrong).

In `StatusMessage` there is a unique enum and a unique `contentType` property in message details but you can have 2 different icons as for example, the **mutual contact** and the **verified**. So in the current `StatusMessage` implementation only one of them can be displayed at the same time, isn't it?

Reply:
Created new task for that: [#541](https://github.com/status-im/StatusQ/issues/541)